apps/app/src/app/core/services/pager.service.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { TestBed } from '@angular/core/testing';
+import { PagerService } from './pager.service';
+
+describe('PagerService', () => {
+  let service: PagerService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [PagerService],
+    });
+    service = TestBed.inject(PagerService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should use defaultNextPage', () => {
```

Comment:
Indiquer dans le titre à quoi est égal defaultNextPage, par exemple : "should use defaultNextPage initialized to 1"